mammoth/opts.py

```diff
@@ -395,6 +370,10 @@ def _add_train_general_opts(parser):
         default=None,
         help='Criteria to use for early stopping.',
     )
+    group.add(
+        '--max_nan_batches', '-max_nan_batches', type=int, default=5,
```

Comment:
what's the logic for the 5 default?

Reply:
honestly, I'd rather have the default zero and give the flag a scary name — some degree of flexibility is useful, but this is essentially an invitation to brute force a buggy impl / poor hparam choices